api/openapi.gen.json

```diff
@@ -414,6 +414,40 @@
       }
     },
     "/reservations/aws": {
```

Comment:
Missing {ID} I think.
`/reservations/aws/{ID}`

Reply:
Right, good catch. Fixing.

Reply:
It did get fixed, and it's working,  but with the latest commit, I am getting the following for `GET /reservations/{ID}`
```
*** iqe_provisioning_api.exceptions.ApiTypeError: Invalid type for variable 'received_data'. Required value type is V1ReservationResponse and passed type was str at ['received_data']
```

Reply:
Ok this is a bug, working on this!